packages/create-llama/helpers/python.ts

```diff
@@ -146,7 +148,26 @@ export const installPythonTemplate = async ({
   const addOnDependencies = getAdditionalDependencies(vectorDb);
   await addDependencies(root, addOnDependencies);
 
-  console.log(
-    "\nPython project, dependencies won't be installed automatically.\n",
-  );
+  // install python dependencies
+  if (isPoetryAvailable()) {
+    console.log(
+      `Installing python dependencies using poetry. This may take a while...`,
+    );
+    const is_installed_success = tryPoetryInstall();
```

Comment:
TS is using camel case for variables ;)
```suggestion
    const installSuccessful = tryPoetryInstall();
```